packages/docs/components/checkbox.md

```diff
@@ -266,82 +266,81 @@ export default {
   <div class="docs-example--rendered">
     <fieldset class="ods-fieldset">
       <legend class="ods-input-legend">Field legend label</legend>
-      <input class="ods-checkbox" type="checkbox" name="checkbox" id="label-1" value="value-1" checked>
-      <label class="ods-checkbox--label" for="label-1">Label 1</label>
-      <input class="ods-checkbox" type="checkbox" name="checkbox" id="label-2" value="value-2">
-      <label class="ods-checkbox--label" for="label-2">Label 2</label>
-      <input class="ods-checkbox" type="checkbox" name="checkbox" id="label-3" value="value-3">
-      <label class="ods-checkbox--label" for="label-3">Label 3</label>
+      <input class="ods-checkbox" type="checkbox" name="example-0" id="example-0-1" value="value-1" checked>
```

Comment:
double check, but i believe `name` should be an array for checkbox groups: `example-0[]`

Reply:
I did a cursory search, ive only seen this kind of thing in PHP apps -- MDN says nothing of it?

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox

Reply:
Perf - thanks for the double-check!